CHANGES.rst

```diff
@@ -1152,6 +1152,9 @@ astropy.units
 - Ensure ``keepdims`` works for taking ``mean``, ``std``, and ``var`` of
   ``Quantity``. [#11198]
 
+- For ``Quantity.to_string()``, ensure that the precision argument is also
```

Comment:
Do I have to move the change log as part of the manual backport?

Reply:
I would think merging is at the time of the release, but am not sure...